tests/test_connection.py

```diff
@@ -1,8 +1,9 @@
 """Connection tests."""
 
+import asyncio
 import asynctest
 import pytest
-from pypck.connection import PchkAuthenticationError, PchkLicenseError
+from pypck.connection import cancel, PchkAuthenticationError, PchkLicenseError
```

Comment:
`cancel` is not used
Adding the tests folder to the linters should be one of the next things we should do. 😏

Reply:
Sounds like a good idea.